lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-redhat-infra_manager.rb

```diff
@@ -1,4 +1,6 @@
 module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Redhat_InfraManager < MiqAeServiceEmsInfra
+    expose :validate_import_vm
+    expose :import_vm
```

Comment:
`import_vm` is just supposed to initiate the import operation so it is not supposed to return anything. I guess by the feature of ruby language it will return the result of the last statement - which is a ovirt ruby sdk entity representing the result of the operation (but in a format totally uninteresting to the caller of this autmation method).